projects/plugins/jetpack/composer.json

```diff
@@ -50,7 +50,8 @@
 		"automattic/jetpack-waf": "@dev",
 		"automattic/jetpack-wordads": "@dev",
 		"automattic/woocommerce-analytics": "@dev",
-		"scssphp/scssphp": "1.12.0"
+		"scssphp/scssphp": "1.12.0",
+		"automattic/jetpack-boost": "*"
```

Comment:
That's really probably not the best way to do it. Ideally the functional parts should be put into packages/boost-core or another package, rather than just dumping the entire plugin into Jetpack's vendor directory.

Reply:
Moving the convo to p1730113871787929-slack-C016BBAFHHS